src/main/java/ch/njol/skript/expressions/ExprSaturation.java

```diff
@@ -39,19 +36,29 @@
 import ch.njol.skript.lang.Expression;
 import ch.njol.skript.lang.ExpressionType;
 import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.util.Getter;
 import ch.njol.util.Kleenean;
+import ch.njol.util.coll.CollectionUtils;
 
 /**
  * Used to set saturation of players. Number is used in case something changes in future...
  * @author bensku
+ * 
+ * Cleaned up by LimeGlass (2.2-dev35)
  */
 @Name("Saturation")
-@Description("The saturation of a player.")
-@Examples({"saturation of player is 20 #Not hungry!"})
-@Since("2.2-Fixes-V10")
+@Description("The saturation of the player(s).")
```

Comment:
Do not use the when you are referring to something for first time.

Reply:
Confused what you mean.

Reply:
Saturation of a player would be proper grammar in this case.